internal/rootcoord/meta_table.go

```diff
@@ -1036,13 +1037,74 @@ func (mt *MetaTable) unlockIsSegmentIndexed(segID typeutil.UniqueID, fieldSchema
 	return exist
 }
 
+func (mt *MetaTable) unlockGetCollectionInfo(collName string) (pb.CollectionInfo, error) {
+	collID, ok := mt.collName2ID[collName]
+	if !ok {
+		collID, ok = mt.collAlias2ID[collName]
+		if !ok {
+			return pb.CollectionInfo{}, fmt.Errorf("collection not found: %s", collName)
```

Comment:
Is it better toi print error info as fmt.Errorf("collection  %s not found", collName) ?

Reply:
Both way make sense to me. But I think maybe `"collection %s not found", collName` is more difficult for me to locate the code.